src/main/java/com/bandwidth/sdk/messaging/MessagingClient.java

```diff
@@ -146,7 +147,9 @@ public MessagingClient build() {
         this.userId = userId;
         this.httpClient = httpClient;
         if (baseUrl != null) {
-            this.BASE_URL = baseUrl;
+            this.baseUrl = baseUrl;
+        } else {
+            this.baseUrl = DEFAULT_BASE_URL;
```

Comment:
this is simple enough I'd probably use the tertiary statement

Reply:
Sounds good with me